### ISSUE TITLE
Track & symlink the precommit hook

### ISSUE THREAD

**Issue by scallyw4g** [scallyw4g/bonsai#18]
The git hooks system is hilariously half-baked so we have to add some gymnastics to the build to actually get the repo hooks onto other peoples' machines

**Comment by PaulSteele34** [comment]
So, create a hooks folder and symlink that to the .git folder equivalents?

**Comment by scallyw4g** [comment]
@PaulSteele34 Exactly.  Basically just busy-work I want to do, but haven't ;)

**Comment by EN1809** [comment]
Yeah, this issue is messing up that error further upstream. So this is a twofer.

**Comment by EN1809** [comment]
Done. I'll create a PR.
